fontir/src/ir.rs

```diff
@@ -60,7 +59,7 @@ pub type DesignSpaceLocation = BTreeMap<String, OrderedFloat<f32>>;
 #[derive(Serialize, Deserialize, Debug, PartialEq)]
 pub struct Glyph {
     pub name: String,
-    pub sources: HashMap<DesignSpaceLocation, GlyphInstance>,
```

Comment:
actually I think for these we don't want UserSpaceLocation. Sources in designspace are always in design-space, not user-space. Instances can avail themselves of being optionally defined in user-space, but sources (aka the designer's "masters") are always design locations because that's what the designer works with, and nobody requested to define sources with user-space location so far.

So I don't think it helps anything to have Glyph::sources keyed by user-space location.

Reply:
I had in mind to leave designspace behind when creating IR and work exclusively in user and normalized space. Is that a bad idea?

Reply:
Reminder for self, per f2f I believe the user of internal coordinates was acceptable here.